expo/gbe/functions.py

```diff
@@ -145,3 +146,24 @@ def available_volunteers(event_start_time):
         if event_start_time in window_range:
             windows.append(window)
     return conf.Volunteer.objects.filter(available_windows__in=windows)
+
+
+def get_events_and_windows(conference):
```

Comment:
Actually, this function doesn't query gbe_volunteerwindows. It takes as an argument an instance of Conference, and it calls a function on that object which returns some objects that have the characteristics that it's looking for. I agree that it violates the Law of Demeter, which is a problem that I was willing to put aside for another day, but the only model it directly queries is scheduler.Event. 

I only point this out because it's an example of the sort of thing that we're going to need to get under control if we're ever going to drag this thing out of the ditch.